sdks/python/generate_pydoc.sh

```diff
@@ -167,7 +167,7 @@ EOF
 
 # Build the documentation using sphinx
 # Reference: http://www.sphinx-doc.org/en/stable/man/sphinx-build.html
-python $(type -p sphinx-build) -v -a -E -q target/docs/source \
+python $(type -p sphinx-build) -v -a -E -j 8 -q target/docs/source \
```

Comment:
Could you use the long forms for these arguments if possible.

Reply:
AFAIK, there isn't a long form for this.
http://www.sphinx-doc.org/en/stable/invocation.html#invocation-of-sphinx-build

Reply:
:/ thank you for checking!